virus/Arbovirus_chinensis.md

```diff
@@ -0,0 +1,83 @@
+```
+## Results
+
+### Comparative Analysis of Host Ecology
+
+```
+Luehdorfia chinensis and Luehdorfia taibai, both endemic to China, share striking similarities in their habitats, behaviors, and appearances. Primarily inhabiting the middle and lower reaches of the Yangtze River in the Qinling Mountains, these species remain poorly understood and are classified as endangered [Guan et al., 2022; Yuan et al., 1998]. Their group-oriented daytime activity is a notable trait, and they primarily feed on dandelions, purpurea, and other viola plants [Yuan et al., 1998].
+
+Ceracris kiangsu, also known as the yellow-spined bamboo locust, is another exclusive Chinese species. Distributed in the southern region of the Yangtze River, this locust significantly impacts bamboo production by feeding on bamboo leaves. Natural predators, including birds, lizards, and mice, contribute to its ecological dynamics. Notably, it exhibits aggregation behavior near human urine-contaminated food due to the strong phagostimulant NaCl and attractant NH4HCO3 [Yu et al., 2011].
+
+Mus musculus, commonly known as the house mouse, belongs to the Rodidae family and is globally distributed, including throughout China. As an omnivore, it consumes seeds, fruits, vegetables, and insects in the wild, with a nocturnal activity pattern. Its role as a carrier of various disease agents is well-documented [Baidu, n.d.].
+
+Meganola albula, or Kent Black Arches, spans the Palearctic region, extending into eastern China, specifically Jiangxi and Sichuan provinces, situated in the middle of the Yangtze River [Boyes et al., 1775; Baidu, n.d.].
+
+The overlapping habitat near the Yangtze River suggests a potential avenue for virus transmission among these species, reflected in the shared presence of the RdRp sequence. The virus discovered may be an insect virus capable of infecting L. chinensis, Luehdorfia taibai, C. kiangsu, and M. albula, all of which are within the ecological range of M. musculus. This ecological connection could explain the presence of the virus in samples from these diverse species. However, the varying activity times—some species being diurnal and others nocturnal—may limit the extent of potential virus transmission due to differences in their active periods.
+```
+
+### Virus Genome {Q3}
+
+![ORF](img/Arbovirus/ORF.png)
```

Comment:
Include a detailed Figure Legend for each Figure. Number Figures used throughout the report in the legend titles.